index.py

```diff
@@ -0,0 +1,4 @@
+from Crawler import Crawler
+
+festify = Crawler()
```

Comment:
I would call this variable something like `crawler` as well. `Festify` is the whole application while this is just an instance of the Crawler class.